src/ssl/cert_validate_message.cc

```diff
@@ -197,6 +197,14 @@ Ssl::CertValidationMsg::getCertByName(std::vector<CertItem> const &certs, std::s
     return NULL;
 }
 
+size_t
+Ssl::CertValidationResponse::MemoryUsedByResponse(const Ssl::CertValidationResponse::Pointer *)
+{
+    // Apparently all the strings and cert memory dont count as being stored.
+    // Presumably they dont use any memory unless something else is also pointing to them.
```

Comment:
I do not understand the intent of this comment. Why "apparently" -- do we have some doubts here? Does this comment document a problem but is missing an XXX prefix? And the "presumably" part does not compute for me at all -- how can a string not use memory?

AFAICT, this entire comment should be replaced with something like

```
// XXX: This math does not account for most of the response size!
```

@chtsanti, do you agree that this size calculation is missing most of the certificate validation response data?

Reply:
"apparently" because you required me to add this code with the reason being that the memory allocations here (outside of Squid) do not use LRU cache bytes. Which really doesn't make sense, but I gave up arguing with you over that.

Reply:
> @chtsanti, do you agree that this size calculation is missing most of the certificate validation response data?

Yep.
The Ssl::CertValidationResponse includes pointers to other objects which can not be computed using just a "sizeof(Ssl::CertValidationResponse)".

Reply:
Glad we are on the same page. My XXX comment [suggestion](https://github.com/squid-cache/squid/pull/30#discussion_r205537717) should work well then.